### DISCUSSION TITLE
MIMIC IV and Optum EHR: patient overlap?

### DISCUSSION THREAD

**Discussion by Kennova8** [MIT-LCP/mimic-code#1755] [MIMIC-IV]
Hello, 

I have built a model to predict missing baseline lab values at ICU admission based on the Optum EHR (features include Demographics, Diagnosis codes, Lab values etc). As an external validation I am now using it to predict baseline lab values in MIMIC IV. However I am worried that there may be some overlap in patients between Optum (100 million patients) and the MIMIC dataset (Currently I have removed patients from the New England region from my Optum training as a precaution). What is the overlap of patients, if any, between Optum EHR and MIMIC? 

Ultimately I don't want to build a model in Optum (training set) and validate on the same patients values in MIMIC (test set). 

Thanks!

**Comment by alistairewj** [comment]
No idea. We're pretty open about our patient inclusion, so you'd have to ask whoever runs Optum and whether they have patients admitted to the BIDMC.
